bundle/tests/quality_monitor/databricks.yml

```diff
@@ -1,19 +1,26 @@
+bundle:
+  name: quality_monitors
+
 resources:
   quality_monitors:
     my_monitor:
-      table_name: "main.test.thing1"
+      table_name: "numbers"
```

Comment:
Is there a reason we don't need to provide `catalog.schema.table` here and can only provide table?

Reply:
Ah, let me just change that back. Note that `table_name` is actually overridden in all targets in the existing example.